backends/tfhe-cuda-backend/cuda/include/integer.h

```diff
@@ -1157,7 +1157,7 @@ template <typename Torus> struct int_sc_prop_memory {
   }
 };
 
-template <typename Torus> struct int_single_borrow_prop_memory {
+template <typename Torus> struct int_overflowing_sub_memory {
```

Comment:
why  name it as `int_overflowing_sub_memory`, this structure  is for a function `unsigned_overflowing_propagate_subtraction_borrow`, old name is really not good, so we had to update it but maybe a different name to specify that it is for `unsigned_overflowing_propagate_subtraction_borrow`

Reply:
also I don't know any other function that's using `unsigned_overflowing_propagate_subtraction_borrow` and can't imagine what else can use it, so maybe it`s fine to name it as `int_overflowing_sub_memory`

Reply:
Yeah this is really the only operation where this is used, so I know it's a bit inconsistent but it seemed easier to read like this to me.